### ISSUE TITLE
Apple Pay: button not appearing on eligible devices.

### ISSUE THREAD

**Issue by Yardco** [woocommerce/woocommerce-gateway-stripe#114]
Am I to early to the Apple Pay party, setting it up wrong or is this an actual bug?

#### What I expected
I expect the apple pay button to appear when a creditcard is set in the wallet app for apple pay and safari is used to check out. I verified the domain in the stripe dashboard and stripe payments work without issues, but the apple pay button simply won't show itself. 

#### What happened instead
Only the regular checkout options are shown, no apple pay button. I tried removing the "display: none;" from "stripe-apple-pay.css",  and the apple pay button then shows (of course), but is not moving forward with processing when clicked, nothing happens.

#### Steps to reproduce the issue
Don't know exactly how to reproduce this, as I expected the apple pay button to show when the device is apple pay enabled and the "enable apple pay" box is checked in settings.

I tried to change the shops base location, default currency etc but none of these things seem to have any effect.

to recap:
- base location of the shop is set to the United Kingdom
- apple pay is enabled in the settings
- secret and live keys are entered, credit card payments are processed without issue
- domain is validated for apple pay in the stripe dashboard

-------------------

- [ ] Issue assigned to next milestone.
- [ ] Issue assigned a priority (will be assessed by maintainers).

**Comment by roykho** [comment]
You need to use a Apple Beta Tester account to test this.
